content/20.treatments.md

```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, many other mundane factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations (generally related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2).
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses.
+The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier stronger response which could reduce viremia and avoid (possibly with the additional support of another therapeutic administered later) the later detrimental immunopathology seen with severe cases; while this is a possibility, additional research is required to assess its validity.
```

Comment:
```suggestion
The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity, possibly with the subsequent administration of an additional therapeutic, might lead to an earlier and stronger response, which could reduce viremia and avoid the later detrimental immunopathology seen in severe COVID-19 cases.
While this is a possibility, additional research is required to assess its validity.
```

Reply:
I need to work on making this sentence clearer while keeping the detail of timing / different roles for different therapeutics and different stages/progression of disease... Either I find a way to write what I wrote more clearly while keeping the points I wanted, or I'll keep the original (less clear) wording. *Reminder* to keep working on making clearer.

Reply:
@yemarshall I was wondering if you're still working on this? I wasn't sure based on your comments this weekend.